tests/conftest.py

```diff
@@ -14,7 +34,8 @@ def testing_images():
     async def _pull():
         docker = Docker()
         required_images = [
-            'alpine:latest', 'redis:latest', 'python:3.6.1-alpine'
+            'alpine:latest', 'redis:latest', 'python:3.6.1-alpine',
+            'busybox:latest',
```

Comment:
The redis image is related to a test, we can remove redis we just need to make the test work in the same way.